packages/app/app/layout.tsx

```diff
@@ -14,6 +14,33 @@ interface RootLayoutProps {
 export default function RootLayout({ children }: RootLayoutProps) {
   return (
     <html lang="en">
+      <head>
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/assets/favicons/apple-touch-icon.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="32x32"
+          href="/assets/favicons/favicon-32x32.png"
+        />
+        <link
+          rel="icon"
+          type="image/png"
+          sizes="16x16"
+          href="/assets/favicons/favicon-16x16.png"
+        />
+        <link rel="manifest" href="/assets/favicons/site.webmanifest" />
+        <link
+          rel="mask-icon"
+          href="/assets/favicons/safari-pinned-tab.svg"
+          color="#F8F8F7"
+        />
+        <meta name="msapplication-TileColor" content="#F8F8F7" />
+        <meta name="theme-color" content="#A2E771" />
+      </head>
```

Comment:
I commented in the header PR, we don't need to set icons like this. Just put the favicon.ico on the root folder.